pkg/parser/epub/positions_service.go

```diff
@@ -1,5 +1,195 @@
 package epub
 
-type PositionsService struct {
-	// TODO
+import (
+	"math"
+
+	"github.com/readium/go-toolkit/pkg/fetcher"
+	"github.com/readium/go-toolkit/pkg/manifest"
+	"github.com/readium/go-toolkit/pkg/service"
+)
+
+/**
+ * Positions Service for an EPUB from its [readingOrder] and [fetcher].
+ *
+ * The [presentation] is used to apply different calculation strategy if the resource has a
+ * reflowable or fixed layout.
+ *
+ * https://github.com/readium/architecture/blob/master/models/locators/best-practices/format.md#epub
+ * https://github.com/readium/architecture/issues/101
+ */
+type EPUBPositionsService struct {
```

Comment:
Since we're in the `epub` package and qualified types are mandatory in Go, I think we can simplify this into `epub.PositionsService`.

```suggestion
type PositionsService struct {
```

`epub.PositionsService` vs `epub.EPUBPositionsService`.

>The importer of a package will use the name to refer to its contents, so exported names in the package can use that fact to avoid repetition.
> https://go.dev/doc/effective_go#package-names